cmd/pinniped-proxy/.rustfmt.toml

```diff
@@ -0,0 +1,68 @@
+max_width = 100
```

Comment:
If these are all defaults, can we not specify them instead? (ie. I think it's much easier to reason about if this file only has the exceptions to the defaults that we want, if any)

Reply:
Yes, I agree. I just wanted to commit all the options to discuss if we should change something else.
However, these options can be retrieved from `rustfmt --print-config default`

> Thanks! I actually thought I had rustfmt running in vscode on save, but obviously not :)

I thought the same, but I broke something in my vscode config and it stopped working. Then I realized I was using the default formatter in vscode and not the rustfmt one.